code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -1203,3 +1203,70 @@
 	glass_name = "glass of töchtaüse syrup"
 	glass_desc = "Not for drinking on its own."
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+
```

Comment:
```suggestion
/datum/reagent/consumable/citrus_control
	name = "citrus control smoothie"
	description = "A powerfully citrus flavored smoothie that can cause delusions of grandeur."
	color = "#FFA500"
	nutriment_factor = 0
	taste_description = "orange and lime"
	glass_icon_state = "orange_and_lime"
	glass_name = "citrus control smoothie"
	glass_desc = "A classic drink, can be very controversial though as it divides the world on those who enjoy the taste and those who do not."
	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
```

Reply:
No headcoder references in my land

Reply:
it should be lemon and lime, in reference to the best maintainer